lib/src/fields/form_builder_checkbox.dart

```diff
@@ -119,7 +119,7 @@ class FormBuilderCheckbox extends FormBuilderField<bool> {
                 value: state.value,
                 onChanged: state.readOnly
                     ? null
-                    : (val) {
+                    : (bool val) {
```

Comment:
🤔 Adding this type is not _necessary_.  It doesn't harm anything, but it doesn't add anything.

Reply:
In some files, the type is specified, that's why I added it.

We should agree on whether to put it or not, and on the name of the variable (val, value, ...).

Reply:
Consistency is good!  When the type says _dynamic_, I see that as a problem that needs to be fixed.  But when the IDE deduces the type, I am OK.